[
    {
        "repo_name": "portainer/agent",
        "CVE_ID": "CVE-2022-24961",
        "Problem_Type": "NVD-CWE-noinfo",
        "Description": "In Portainer Agent before 2.11.1, an API server can continue running even if not associated with a Portainer instance in the past few days.",
        "URL": "https://github.com/portainer/agent/pull/225/commits/a66977c76043fcff4a8f69c4b65988272d27c01f",
        "Tag": "['Patch', 'Third Party Advisory']",
        "fix_commit_hash": "e39fe0173fc4e63da02c56d10ca11f9967ed2e07",
        "inducing_commit_hash_pyszz": [
            "2ae25321db7113072183815bd2ab13b87cae9633"
        ],
        "inducing_commit_hash_pd": [
            "2ae25321db7113072183815bd2ab13b87cae9633",
            "0acb11b93647cd4078394072bed112a815f00176"
        ],
        "matched": [
            "2ae25321db7113072183815bd2ab13b87cae9633"
        ]
    }
]